### PR TITLE
all: update copyright span to 2019-2021

### PR DESCRIPTION
**Pull request by serkonda7**

### CODE REVIEW

**Comment by serkonda7** [comment]
@danieldaeschle have you any idea why the fuzzing job fails?

**Comment by danieldaeschle** [comment]
seems like a bug in the parser

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019-2020 Alexander Medvednikov
+Copyright (c) 2019-2021 Alexander Medvednikov
```
Comment (danieldaeschle) [inline LICENSE:3]: should we have a timespan? that would need to be updated every year

Comment (serkonda7) [reply LICENSE:3]: @medvednikov

Comment (serkonda7) [reply LICENSE:3]: Else we would need to update it every year too and just replace `(c) 2020` with `(c) 2021`, etc.

**Comment by JalonSolov** [comment]
```
Form of notice for visually perceptible copies

The notice for visually perceptible copies should contain all three elements described below. They should appear together or in close proximity on the copies.

    The symbol © (letter C in a circle); the word “Copyright”; or the abbreviation “Copr.”
    The year of first publication. If the work is a derivative work or a compilation incorporating previously published material, the year date of first publication of the derivative work or compilation is sufficient. Examples of derivative works are translations or dramatizations; an example of a compilation is an anthology. The year may be omitted when a pictorial, graphic, or sculptural work, with accompanying textual matter, if any, is reproduced in or on greeting cards, postcards, stationery, jewelry, dolls, toys, or useful articles.[1]
    The name of the copyright owner, an abbreviation by which the name can be recognized, or a generally known alternative designation of owner.

Example: © 2012 Jane Doe
```
and
```
Once in a while you might see what looks like a date range instead of a single year of first publication, such as: © Epiphany Law Pty Ltd, 2006-2016. This is appropriate for ‘documents’ such as web pages or software programs which are constantly being revised and updated incrementally. The ‘start’ year should be the date of the oldest content still contained in the work, with the ‘end’ year being the last year that the work was revised.
```

**Comment by JalonSolov** [comment]
Found it...
```
Because the © symbol is not available on typical typewriters or in ASCII, it has long been common to approximate this symbol with the characters (C) (C in parentheses), a practice that has been accepted by the U.S. Copyright Office under both the 1909 and 1976 U.S. Copyright Acts
```
So it should be `(C)` instead of `(c)` to be legal, otherwise have to go to UNICODE for ©:
```
The character is mapped in Unicode as U+00A9 © COPYRIGHT SIGN.[17] Unicode also has U+24B8 Ⓒ CIRCLED LATIN CAPITAL LETTER C and U+24D2 ⓒ CIRCLED LATIN SMALL LETTER C,[18] which have an appearance similar to the character.
```

**Comment by serkonda7** [comment]
@JalonSolov could you give a link please, where this information is from.

**Comment by JalonSolov** [comment]
https://en.wikipedia.org/wiki/Copyright_notice

and

https://epiphany.law/articles/copyright/copyright-dates-single-year-or-range

**Pull request merged**